src/pypnusershub/routes.py

```diff
@@ -250,6 +250,27 @@ def login():
         msg = json.dumps({"login": False, "msg": repr(e)})
         return Response(msg, status=403)
 
+@routes.route("/public_login", methods=["POST"])
+def public_login():
+    if current_app.config.get("PUBLIC_ACCESS")["ENABLE_PUBLIC_ACCESS"]:
```

Comment:
Si ``PUBLIC_ACCESS`` n’existe pas dans la configuration (par exemple avec TaxHub ou UsersHub, qui utilise cette module également), alors ``current_app.config.get("PUBLIC_ACCESS")`` vaudra ``None`` et accéder à ``ENABLE_PUBLIC_ACCESS`` provoquera une erreur.
```suggestion
    if current_app.config.get("PUBLIC_ACCESS", {}).get("ENABLE_PUBLIC_ACCESS"):
```

(Par ailleurs, détail mais inverser la logique (si pas d’accès publique, on raise Forbdiden en début de fonction) permet d’éviter un niveau d’indentation à l’ensemble du code.)